app/Http/Controllers/UserController.php

```diff
@@ -97,4 +100,45 @@ public function logout(): JsonResponse
         Auth::logout();
         return new JsonResponse(new stdClass());
     }
+
+    /**
+     * @throws Throwable
```

Comment:
może coś bardziej szczegółowo o typie Exceptiona? :D

jak się daje Najpierw atrybuty potem adnotacje, to phpstorm pozwala, żeby były w tej samej linii `()] /**`, w sumie nie wiem jaka jest zalecana kolejność